zrml/neo-swaps/src/math.rs

```diff
@@ -226,12 +256,28 @@ mod detail {
         Some((liquidity, reserves))
     }
 
+    /// Calculate e^(x/b) − 1 + e^(−r_i/b).
+    pub(super) fn calculate_buy_ln_argument_fixed(
+        reserve: Fixed,
+        amount_in: Fixed,
+        liquidity: Fixed,
+    ) -> Option<Fixed> {
+        let exp_x_over_b: Fixed = exp(amount_in.checked_div(liquidity)?, false).ok()?;
+        let r_over_b = reserve.checked_div(liquidity)?;
+        let exp_neg_r_over_b = if r_over_b < Fixed::from_str(EXP_OVERFLOW_THRESHOLD).ok()? {
```

Comment:
In case parsing `EXP_OVERFLOW_THRESHOLD` from bits is applied:
```suggestion
        let exp_neg_r_over_b = if r_over_b < EXP_OVERFLOW_THRESHOLD {
```